front-end-web/src/Components/Pages/SelectionPage.tsx

```diff
@@ -1,18 +1,62 @@
+import { Checkbox, List, ListItem, Button } from "@material-ui/core";
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { getUserCalendars } from "../../scripts";
 
 /**
  * TODO: Add Documentation
  */
-const SelectionPage: React.FunctionComponent<Props> = ({
-  state,
-  handlers,
-  classes
+interface SelectionProps {
+  calendars: calendar[];
+  selectedCalendars: boolean[];
+  handleSelect: Function;
+  handleSubmit: Function;
+  setStage: Function;
+}
+
+const SelectionPage: React.FunctionComponent<SelectionProps> = ({
+  calendars,
+  selectedCalendars,
+  handleSelect,
+  handleSubmit,
+  setStage
```

Comment:
I absolutely love that you are already breaking it down into just the parameters you need, and this is definitely what we will go to in the future. Unfortunately, you're about 4 to 6 weeks ahead of schedule, and this is causing issues with the main page. Would you mind putting this back for Tuesday's demo, and we'll add your changes back in later?